docs/network-flow-visibility.md

```diff
@@ -202,30 +203,69 @@ helm install theia build/charts/theia -n flow-visibility --create-namespace \
   --set=clickhouse.storageSize="2Gi"
 ```
 
-The ClickHouse TCP service is used by the Flow Aggregator. If you have changed the TCP port,
-please update the `databaseURL` following
-[Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1).
+The ClickHouse TCP Service is used by the Flow Aggregator. If you have changed
+the TCP port, please update the `clickHouse.databaseURL` in the Flow Aggregator
+Helm Chart values.
+
+The ClickHouse credentials are used in the Flow Aggregator. If you have changed
+the ClickHouse credentials, please update the `clickHouse.connectionSecret` in the
+Flow Aggregator Helm Chart values.
+
+Please refer to the Flow Aggregator Helm Chart
+[README](https://github.com/antrea-io/antrea/blob/main/build/charts/flow-aggregator/README.md)
+and [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
+to learn more about ClickHouse configuration options in Flow Aggregator.
+
+##### ClickHouse Cluster
+
+In v0.1, Theia deploys ClickHouse in a non-cluster mode, which cannot switch to
+ClickHouse cluster directly. You need to uninstall ClickHouse, delete related
+data if you use PersistenVolume and redeploy it when upgrading or downgrading
+between v0.1 and v0.2.
+
+Starting with v0.2, Theia supports ClickHouse database cluster with data replication,
+and deprecates the non-cluster mode. A ClickHouse cluster consists of one or more
+shards. Shards refer to the servers that contain different parts of the data. You
+can deploy multiple shards to scale the cluster horizontally. Each shard consists
+of one or more replica hosts. Replicas indicates storing the same data on multiple
+Pods to improve data availability and accessibility. You can deploy multiple
+replicas in a shard to ensure reliability.
+
+By default, Theia deploys a 1-shard-1-replica ClickHouse cluster and a 1-replica
+ZooKeeper. To scale the ClickHouse cluster, please set `clickhouse.cluster.shards`
+and `clickhouse.cluster.replicas` per your requirement.
+
+ClickHouse uses [Apache ZooKeeper](https://zookeeper.apache.org/) for storing
+replicas meta information. When scaling the ClickHouse cluster, please set
+`clickhouse.cluster.installZookeeper.replicas` to at lease 3 to ensure fault
+tolerance. Each replica is expected to be deployed on a different Node. To
+use a customized ZooKeeper cluster, please refer to the
```

Comment:
more than "customized ZK cluster", shall we consider "to leverage a user-provided ZK cluster..."?

Reply:
It's more accurate. Update. Thanks!